[AMSDK-9052] Alpha Mobile Platform extension docs for schema and auto-collected data

kevinlind opened the pull request:
Additional documentation for Alpha Mobile Platform extension
- Schema used in Alpha release
- Auto-collected data in Mobile Platform Extension
- Update Summary.md to link to pages (follow format in PR #209)

kevinlind commented on SUMMARY.md:
```diff
@@ -106,3 +106,9 @@
 
+## ALPHA
+
+- [Experience Platform Extension](alpha/experience-platform-extension/README.md)
```
This 'README.md' is not in this PR, but is being added in PR #209

kevinlind commented on alpha/experience-platform-extension/schema-creation.md:
```diff
@@ -0,0 +1,191 @@
+The sample XDM schema for the Experience Platform Mobile extension extends from the [XDM ExperienceEvent]() class, which is a time-series based class and captures the state of the system when an event, or events, occurred. Three mixins are also included in the schema: 
+
+- [ExperienceEvent Application Details](https://github.com/adobe/xdm/blob/1c22180490558e3c13352fe3e0540cb7e93c69ca/docs/reference/context/experienceevent-application.schema.md) - data related to the application generating or targeted by the event
+- [ExperienceEvent Environment Details](https://github.com/adobe/xdm/blob/1c22180490558e3c13352fe3e0540cb7e93c69ca/docs/reference/context/experienceevent-environment-details.schema.md) - data related to the device, location, and surrounding situation
```
I changed "situation" to "environment". I think environment is better than situation, but if there's a better word let me know. The usage here is "the situation, or environment, at the time the event occurred, such as operating system, mobile carrier, and IP address".

rekrishn commented on alpha/experience-platform-extension/auto-collected-data.md:
```diff
@@ -0,0 +1,70 @@
+
+## Device
+
+[Device](https://github.com/adobe/xdm/blob/1c22180490558e3c13352fe3e0540cb7e93c69ca/docs/reference/context/device.schema.md) is included in the [Experience Cloud Environment details](https://github.com/adobe/xdm/blob/1c22180490558e3c13352fe3e0540cb7e93c69ca/docs/reference/context/experienceevent-environment-details.schema.md) mixin and details data about the mobile device. The following are the d ...
```
Please reword the last sentencer as follows:
"Here are the device properties the Experience Platform Mobile extension automatically adds to each Platform event:" --> please replace the period with a colon.

rekrishn commented on alpha/experience-platform-extension/schema-creation.md:
```diff
@@ -0,0 +1,191 @@
+
+### Product list items
+
+The `ProductListItemsItem` object includes details for a product that was selected by the customer. A list of  `ProductListItemsItem` objects can be used to identity the products with which the customer has interacted.
```
Please reword the 2nd sentence to:
"The following `ProductListItemsItem` objects can be..."

rekrishn commented on alpha/experience-platform-extension/auto-collected-data.md:
```diff
@@ -0,0 +1,70 @@
+
+## Environment
+
+[Environment](https://github.com/adobe/xdm/blob/1c22180490558e3c13352fe3e0540cb7e93c69ca/docs/reference/context/environment.schema.md) is included in the  [ExperienceCloud Environment Details](https://github.com/adobe/xdm/blob/1c22180490558e3c13352fe3e0540cb7e93c69ca/docs/reference/context/experienceevent-environment-details.schema.md) mixin and includes information about the surrounding environm ...
```
Please reword the last sentence as follows:
"Here are the device properties the Experience Platform Mobile extension automatically adds to each Platform event:" --> please change from a period to a colon at the end."

rekrishn commented on alpha/experience-platform-extension/schema-creation.md:
```diff
@@ -1,39 +1,39 @@
+Each action object that is defined in the `Commerce` object takes a value, typically set to 1, which indicates that action occurred.
 
-- CartAbandons - A product list has been identified as no longer accessible (e.g purchasable) by the user.
+- CartAbandons - A product list has been identified as no longer accessible (for example purchasable) by the user.
```
please add a comma between "example" and "purchasable".

rekrishn commented on alpha/experience-platform-extension/auto-collected-data.md:
```diff
@@ -0,0 +1,70 @@
+
+The current screen orientation, and the options are portrait and landscape.
+
+#### Type
```
@emdobrin, a table is a great idea for both of the sections...provides a break from all of the text too.

kevinlind commented:
@rekrishn , @emdobrin Thanks for the review and suggestions. I've updated the docs based on the latest feedback and changed the format of auto-collected-data.md to use tables. I also added another section to auto-collected-data to include the two auto-generated properties ID and Timestamp for the Platform Event object.

emdobrin commented on alpha/experience-platform-extension/schema-creation.md:
```diff
@@ -0,0 +1,191 @@
+
+### Product list items
+
+The `ProductListItemsItem` object includes details for a product that was selected by the customer. The following `ProductListItemsItem` objects can be used to identity the products with which the customer has interacted.
```
Should this be `for a product that was selected by the user` and `which the user has interacted`?

rekrishn replied:
that works for me too @kevinlind and @emdobrin

rekrishn commented on alpha/experience-platform-extension/auto-collected-data.md:
```diff
@@ -16,7 +16,7 @@ The Experience Cloud Identifier (ECID)  that is generated by the Adobe Experiece
 ## Device
 
-[Device](https://github.com/adobe/xdm/blob/1c22180490558e3c13352fe3e0540cb7e93c69ca/docs/reference/context/device.schema.md) is included in the [Experience Cloud Environment details](https://github.com/adobe/xdm/blob/1c22180490558e3c13352fe3e0540cb7e93c69ca/docs/reference/context/experienceevent-environment-details.schema.md) mixin and details data about the mobile device. The following are the d ...
+[Device](https://github.com/adobe/xdm/blob/1c22180490558e3c13352fe3e0540cb7e93c69ca/docs/reference/context/device.schema.md) is included in the [Experience Cloud Environment details](https://github.com/adobe/xdm/blob/1c22180490558e3c13352fe3e0540cb7e93c69ca/docs/reference/context/experienceevent-environment-details.schema.md) mixin and details data about the mobile device. Here are the device pro ...
```
In the 2nd sentence, please add "that" between "properties" and "the"

rekrishn commented on alpha/experience-platform-extension/auto-collected-data.md:
```diff
@@ -52,7 +52,7 @@ The type of device that is being tracked. The current options are phone and tabl
 ## Environment
 
-[Environment](https://github.com/adobe/xdm/blob/1c22180490558e3c13352fe3e0540cb7e93c69ca/docs/reference/context/environment.schema.md) is included in the  [Experience Cloud Environment details](https://github.com/adobe/xdm/blob/1c22180490558e3c13352fe3e0540cb7e93c69ca/docs/reference/context/experienceevent-environment-details.schema.md) mixin and includes information about the surrounding environ ...
+[Environment](https://github.com/adobe/xdm/blob/1c22180490558e3c13352fe3e0540cb7e93c69ca/docs/reference/context/environment.schema.md) is included in the  [Experience Cloud Environment details](https://github.com/adobe/xdm/blob/1c22180490558e3c13352fe3e0540cb7e93c69ca/docs/reference/context/experienceevent-environment-details.schema.md) mixin and includes information about the surrounding environ ...
```
In the 2nd sentence, please add "that" between "properties" and "the"

rekrishn commented on alpha/experience-platform-extension/schema-creation.md:
```diff
@@ -1,6 +1,6 @@
 # XDM Schema
 
-Experience Data Model (XDM) schemas are a set a rules that defines and validates the customer experience data format in Adobe Experience Platform. XDM schemas are composed of one class and zero or more mixins. For more information on XDM Schemas, see [XDM System overview](https://www.adobe.io/apis/experienceplatform/home/xdm/xdmservices.html#!api-specification/markdown/narrative/technical_overvie ...
+Experience Data Model (XDM) schemas are a set a rules that define and validate the customer experience data format in Adobe Experience Platform. XDM schemas are composed of one class and zero or more mixins. For more information about XDM Schemas, see [XDM System overview](https://www.adobe.io/apis/experienceplatform/home/xdm/xdmservices.html#!api-specification/markdown/narrative/technical_overvi ...
```
- In the 1st sentence, please replace "a set a" with "a set of"
- in the 3rd sentence, please replace "information on" with "information about"

rekrishn commented on alpha/experience-platform-extension/auto-collected-data.md:
```diff
@@ -2,9 +2,18 @@
 
+## Platform event
+
+The [Experience Event](https://github.com/adobe/xdm/blob/master/docs/reference/context/experienceevent.schema.md) is a time-based record of an event. The XDM Experience Event schema is the top-level class for all Experience events and contains information about the event, device, environment, and the identity of the user. Here are the properties the Experience Platform Mobile extension automatica ...
```
In the 2nd sentence, please add "that" between "properties" and "the"

rekrishn commented on alpha/experience-platform-extension/auto-collected-data.md:
```diff
@@ -2,9 +2,18 @@
 ## Identity
 
-The [IdentityMap](https://github.com/adobe/xdm/blob/1c22180490558e3c13352fe3e0540cb7e93c69ca/docs/reference/context/identitymap.schema.md) defines a map that contains end-user identities.
+The [IdentityMap](https://github.com/adobe/xdm/blob/1c22180490558e3c13352fe3e0540cb7e93c69ca/docs/reference/context/identitymap.schema.md) defines a map that contains end-user identities. Here are the identity properties the Experience Platform Mobile extension automatically adds to each Platform event:
```
In the 2nd sentence, please add "that" between "properties" and "the"

emdobrin commented:
@kevinlind my changes were merged into experience-platform-staging, make sure you rebase before merging.

kevinlind commented:
Thanks @rekrishn and @emdobrin! I've pushed the latest changes.
